### PR TITLE
Update Scala versions to 2.12.11

### PR DESCRIPTION
**Pull request by ijuma**

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=develop&zinc=pull/751/head&io=develop&librarymanagement=develop&util=develop&website=develop&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

File: project/Dependencies.scala [hunk 1]
```diff
@@ -6,7 +6,7 @@ object Dependencies {
 
   val scala210 = "2.10.7"
   val scala211 = "2.11.12"
-  val scala212 = "2.12.10"
+  val scala212 = "2.12.11"
   val scala213 = "2.13.1"
```
Comment (ijuma) [inline project/Dependencies.scala]: Thanks for the various fixes. Scala 2.13.2 has been released this week, do we want to update that too?

Comment (dwijnand) [reply project/Dependencies.scala]: Yeah, good thinking.

**Review by dwijnand — APPROVED** [review]

**Review by eed3si9n — APPROVED** [review]
Thanks @ijuma and @dwijnand!

**Pull request merged**